doc/tutorials/using_partition_mode_on_up2.rst

```diff
@@ -2,154 +2,312 @@
+   editing the source code later.
 
-  .. code-block:: none
+   .. code-block:: none
+     :emphasize-lines: 1,9,10,14,22
+
+     $ sudo lspci -vv
+     ...
+     00:12.0 SATA controller: Intel Corporation Device 5ae3 (rev 0b) (prog-if 01 [AHCI 1.0])
+         Subsystem: Intel Corporation Device 7270
+         Control: I/O+ Mem+ BusMaster+ SpecCycle- MemWINV- VGASnoop- ParErr- Stepping- SERR- FastB2B- DisINTx+
+         Status: Cap+ 66MHz+ UDF- FastB2B+ ParErr- DEVSEL=medium >TAbort- <TAbort- <MAbort- >SERR- <PERR- INTx-
+         Latency: 0
+         Interrupt: pin A routed to IRQ 123
+         Region 0: Memory at 91514000 (32-bit, non-prefetchable) [size=8K]
+         Region 1: Memory at 91537000 (32-bit, non-prefetchable) [size=256]
+         Region 2: I/O ports at f090 [size=8]
+         Region 3: I/O ports at f080 [size=4]
+         Region 4: I/O ports at f060 [size=32]
+         Region 5: Memory at 91536000 (32-bit, non-prefetchable) [size=2K]
+     ...
+     00:15.0 USB controller: Intel Corporation Device 5aa8 (rev 0b) (prog-if 30 [XHCI])
+         Subsystem: Intel Corporation Device 7270
+         Control: I/O- Mem+ BusMaster+ SpecCycle- MemWINV- VGASnoop- ParErr- Stepping- SERR- FastB2B- DisINTx+
+         Status: Cap+ 66MHz- UDF- FastB2B+ ParErr- DEVSEL=medium >TAbort- <TAbort- <MAbort- >SERR- <PERR- INTx-
+         Latency: 0
+         Interrupt: pin A routed to IRQ 122
+         Region 0: Memory at 91500000 (64-bit, non-prefetchable) [size=64K]
+
+#. Clone the ACRN source code [1]_ and configure the build options with
+   ``make menuconfig`` command:
 
-     cd ~/acrn-hypervisor/hypervisor
-     make menuconfig
+   .. code-block:: none
+     :emphasize-lines: 1,3
 
-  Select entry ``Base address of serial MMIO region`` and press ``Enter``.
-  Then we will see a prompt msgbox with title ``Value for `Base address of serial MMIO region` (hex)``.
-  Please type the serial base mmio address of your UP2 board and press ``Enter``.
+     $ git clone https://github.com/projectacrn/acrn-hypervisor.git
+     $ cd acrn-hypervisor/hypervisor
+     $ make menuconfig
 
-* Configure the PCI info about devices assigned to vm.
+   Set the ``Hypervisor mode`` option to ``Partition mode``, and depending
+   on the serial port you are using, enter its MMIO address to the configuration
+   menu as shown in below screenshot. Finally, save the configuration.
 
-  For the current code, we can only configure it by modifying the ``vpci_vdev_arrary``
-  structure (one per vm) in source code file ``hypervisor/partition/vm_description.c``.
+   .. figure:: images/menuconfig-partition-mode.png
+      :align: center
 
-  Here is an reference patch for configuring vm1's assgined PCI devices based on
-  UP2 board. This patch can also apply to other vms you try to configure.
+#. Configure the PCI device info for each VM
 
-  .. code-block:: none
+   In current implementation, PCI devices that are available to the privileged VMs
```

Comment:
Unless there's a plan to change this, I'd delete the ``In current implementation, ``

Reply:
It seems to me one of the developer is developing a logic to host the **board** based vm_description.c file, I'd keep this until the new logic merged to the repo.

Reply:
I've verified the commit #1861, the path to ``vm_description.c`` has been updated. Remove ``In current implementation,`` per your comment.